diff --git a/dist/index.js b/dist/index.js
index 53d5b41865172e8409d1e9c344729526b6a44338..eaf7191be6281a517bce00939e0ac8652c2915ae 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -99,12 +99,12 @@ const rehypeCallouts = (options) => {
             // modify the blockquote element
             // @ts-expect-error (Type '"div" | "details"' is not assignable to type '"blockquote"')
             node.tagName = collapsable ? 'details' : 'div';
+            const revisedType = type.toLowerCase();
             node.properties.className = [
                 'callout',
+                `callout-${revisedType}`,
                 collapsable && 'callout-collapsible',
             ];
-            const revisedType = type.toLowerCase();
-            node.properties.style = generateStyle(expandedCallouts[revisedType].color);
             // update hast
             node.children = [
                 h(collapsable ? 'summary' : 'div', {
